### DISCUSSION TITLE
Commit count in stats card seems to be incorrect

### DISCUSSION THREAD

**Discussion by astanwar99** [anuraghazra/github-readme-stats#3059] [Q&A] [answered]
It looks like my total commits in stats card is incorrect. I am using my own vercel instance.

**Card URL:**
[![Ashwani's GitHub stats](http://github-readme-stats-git-master-astanwar99.vercel.app/api?username=astanwar99&count_private=true&show_icons=true&theme=radical)](https://github.com/astanwar99/github-readme-stats)

```
[![Ashwani's GitHub stats](http://github-readme-stats-git-master-astanwar99.vercel.app/api?username=astanwar99&count_private=true&show_icons=true&theme=radical)](https://github.com/astanwar99/github-readme-stats)
```

**GraphQL Explorer** (From this discussion: https://github.com/anuraghazra/github-readme-stats/discussions/2959)

![image](https://github.com/anuraghazra/github-readme-stats/assets/30383121/f7829ea5-de8c-4f8e-99fc-6deabd622503)

I was also wondering if its possible to include contributions that have been made in an organisation. If a user is member or owner of organisation - in either case.

Thanks!

**Comment by qwerty541** [comment, marked as answer]
> It looks like my total commits in stats card is incorrect. I am using my own vercel instance.

This way to check if commits count is correct no more actual since we don't use `restrictedContributionsCount` value during stats calculation. See: 

https://github.com/anuraghazra/github-readme-stats/blob/master/src/fetchers/stats-fetcher.js#L47-L50

`&count_private=true` query option also no more supported, so you can remove it. `totalCommitsContributions` value should contain all your commits (public/private) if you correctly set permissions for you GitHub Personal Access Token (PAT).

> I was also wondering if its possible to include contributions that have been made in an organisation. If a user is member or owner of organisation - in either case.

Currently this feature is not officially supported, but you can find several pull requests with draft implementations and deploy them on your own Vercel instance.

**Reply by zeyuanyin** [reply]
Hi @qwerty541 , please tell me which permissions I need to enable for `&count_private=true`. Thanks a lot!

![image](https://github.com/anuraghazra/github-readme-stats/assets/51396847/97272979-312e-4957-aa94-000a556d1fbc)
![image](https://github.com/anuraghazra/github-readme-stats/assets/51396847/dcc55935-9e6b-4815-816d-634899d16737)

**Reply by qwerty541** [reply]
@zeyuanyin Looks like you are using "Fine-grained" token which currently in beta. I have never used them. I'm using classic token, here is my permissions:

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/32f124bb-52ce-4155-a95c-4e301fd369e9)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/6893d950-935e-424e-85ce-6137ce984a6a)

I recommend you to migrate on classic token.

Personally i have not tested them, but other users already comlained that fine-grained does not work. Here is an example: https://github.com/anuraghazra/github-readme-stats/discussions/2911

**Reply by zeyuanyin** [reply]
@qwerty541 Thanks for your suggestion. After opening the repo and user permissions on the classic token, `&count_private=true` works well.

**Reply by rickstaa** [reply]
The new permissions should also work. However, I still need to determine which settings to enable the cards to work. Nevertheless, as long as the classic permission tokens are supported, this will be the recommended way since they can be set not to expire 👍🏻.
